index.html

```diff
@@ -61,7 +85,7 @@ <h1 class="light_on_dark">Not Sure Where to Start?</h1>
 
     <div>
 
-      <h1>Check Out These Schools</h1>
+      <h1>Check Out <br>These Schools</h1>
```

Comment:
Yep. The many <br> in these are because I'm not sure that the text will stay as-is and so I didn't want to spend time until we knew the final words/design implementation.